src/AddIns/Uno.UI.Lottie/WasmScripts/uno-lottie.d.ts

```diff
@@ -16,7 +15,7 @@ declare namespace Uno.UI {
         private static _player;
         private static _runningAnimations;
         private static _numberOfFrames;
-        static setAnimationProperties(newProperties: LottieAnimationProperties, animationData?: AnimationData): string;
+        static setAnimationProperties(newProperties: LottieAnimationProperties): string;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class method 'setAnimationProperties' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5499310)